src/xdb_transform.erl

```diff
@@ -268,6 +271,14 @@ build_repo_fun(Repo, Adapter, Mod, Fun, Args) ->
   Body = build_repo_fun_body(Fun, "~p, ~p", Args),
   ?Q(text(Body, [Mod, Repo, Adapter])).
 
+%%@private
+build_repo_fun_spec(_, rollback, _) ->
+  ?Q(lists:flatten(["-spec ", "rollback", "(any()) -> no_return()."]));
```

Comment:
This spec is required because new Dialyzer is very strict for `no_return()` functions